topics/transcriptomics/tutorials/mirna-target-finder/tutorial.md

```diff
@@ -0,0 +1,726 @@
+---
+layout: tutorial_hands_on
+
+title: 'miRNA target indentification in Arabidopsis thaliana'
+zenodo_link: 'https://zenodo.org/record/4606701'
+tags:
+    - miRNA
+    - plants
+questions:
+- Which miRNAs are upregulated in response to brassinosteroids?
+- Which genes are potential target of brassinosteroid-induced miRNAs?
+objectives:
+- Perform miRNA differential expression analysis
+- Understand quasi-mapping-based Salmon method for quantifying the expression of transcripts using RNA-seq data
+- Idenfity potential miRNAs involved in brassinosteroid-mediated regulation networks
+time_estimation: '2H'
+key_points:
+- We learned to perform a miRNA differential expression analysis
+- We used a quasi-mapping approach for quantifiying gene expression
+- We learned how to identify potential miRNA targets
```

Comment:
they key points should be phrased more like conclusions, so instead of of "we learned how to identify miRNA targets" something more like "Doing X and Y enables identification of potential miRNA targets" , or  "A Quasi-mapping approach is useful for/because X" ..basically ask yourself what are the  3-10 things you hope people will learn/remember from this tutorial,

Reply:
Nice, thanks!